--- Makefile	2015-01-12 11:59:37.000000000 +0000
+++ Makefile.new	2015-01-12 11:59:51.000000000 +0000
@@ -14,7 +14,6 @@
 
 RUN_DEPENDS=	dmidecode>0:${PORTSDIR}/sysutils/dmidecode \
 		sudo>0:${PORTSDIR}/security/sudo \
-		gpart>0:${PORTSDIR}/sysutils/gpart \
 		${PYTHON_PKGNAMEPREFIX}boto>0:${PORTSDIR}/devel/py-boto \
 		${PYTHON_PKGNAMEPREFIX}Jinja2>0:${PORTSDIR}/devel/py-Jinja2 \
 		${PYTHON_PKGNAMEPREFIX}prettytable>0:${PORTSDIR}/devel/py-prettytable \
